specification/langRef/ditaval/ditaval-prop.dita

```diff
@@ -98,10 +98,10 @@
                 <dt>passthrough</dt>
                 <dd>Indicates that the content is included in the output
                   and that the attribute value is preserved. This enables
-                  further processing by a runtime engine. The attribute
+                  further processing by a runtime engine.<!-- The attribute
```

Comment:
Per editorial meeting discussion, the extra third sentence here does not actually add anything apart from confusion; commenting it out.